docs/reference/index.md

```diff
@@ -1,5 +1,11 @@
 # Welcome to HACL Packages' documentation!
 
+HACL Packages provides ready-to-use crypto algorithms based on the formally verified [HACL\* crypto distribution].
+
+This manual describes the HACL\* and EverCrypt C APIs. The HACL\* API provides a lightweight and direct interface to all supported algorithms and their different implementations. To use the most efficient implementation, you must be aware of the platform you intend to run HACL\* on.
+
+EverCrypt, on the other hand, is a cryptographic *provider*. The EverCrypt API provides a unified interface to all supported algorithms. It always selects the most efficient implementation that is available at runtime.
```

Comment:
Well, it's not for *all supported algorithms*. Some are missing and others only exist in evercrypt 😬 But fine to keep as is if you can't think of a better wording.

Reply:
Let's keep it as is. As long as we make it clear later that something doesn't exist, its okay. I think it is better to keep it simple here.